src/MongoDB.Driver.Core/Core/Operations/RetryabilityHelper.cs

```diff
@@ -153,15 +170,9 @@ public static bool IsRetryableWriteException(Exception exception)
                         case ServerErrorCode.HostUnreachable:
                         case ServerErrorCode.NetworkTimeout:
                         case ServerErrorCode.SocketException:
+                        case ServerErrorCode.ExceededTimeLimit:
                             return true;
                     }
-
-                    var message = writeConcernError.GetValue("errmsg", null)?.AsString;
```

Comment:
this is no longer required according to this doc line: 
>Change the definition of "retryable write" to be based on the RetryableWriteError label. Stop requiring drivers to parse errmsg to categorize retryable errors for pre-4.4 servers

https://github.com/mongodb/specifications/blob/44d71db7bff57b95cd65b554cf0aedb5159e58d9/source/retryable-writes/retryable-writes.rst#changes

Reply:
according to the [spec](https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst#why-did-an-earlier-version-of-the-spec-require-the-driver-to-parse-the-error-message-to-categorize-retryable-errors), the behavior should be the same as before:
>Retryable writes are only available on MongoDB versions 3.6 and newer. These server versions can be relied on to consistently return the proper error codes. Thus, it is sufficient that the driver check errors for the appropriate error codes or error labels to determine whether they are retryable without performing any error message parsing.